tools/sof-dump-status.py

```diff
@@ -339,6 +340,17 @@ def _covertType(type_lst):
             exit(0)
         print(card_info['longname'])
         exit(0)
+    if ret_args.get('community') is True:
+        vender_list = ["AAEON", "Google"]
+        board_list = ["UP-APL01"]
+
+        sysinfo.loadDMI()
+        board = sysinfo.dmi["board_name"]
+        vendor = sysinfo.dim["board_vendor"]
```

Comment:
I suggest convert all string to lower/upper to compare to avoid some mistake

Reply:
I don't think this is needed, these string used here is from Linux Kernel, and the kernel will use the string from BIOS, all 3 places has the same form. btw, there is no case transform in the kernel for this purpose.  the string got by sof-dump-status.py is always the same as the kernel.

Reply:
Please add some inline comments here to the file to help others keep this part synced with kernel.